discovery/gossiper.go

```diff
@@ -1348,10 +1373,10 @@ func (d *AuthenticatedGossiper) processNetworkAnnouncement(nMsg *networkMsg) []n
 			// itself to the database so we can fetch it later when
 			// gossiping with other nodes.
 			proof = &channeldb.ChannelAuthProof{
-				NodeSig1:    msg.NodeSig1,
-				NodeSig2:    msg.NodeSig2,
-				BitcoinSig1: msg.BitcoinSig1,
-				BitcoinSig2: msg.BitcoinSig2,
+				NodeSig1Bytes:    msg.NodeSig1.ToSignatureBytes(),
```

Comment:
Why not make `channeldb` take `lnwire.Sig`s?

Reply:
We _could_ have done it, but this maintains the existing serialization (which is how it's serialized everywhere in Bitcoin).